docs/pages/connect-your-client/tsh.mdx

```diff
@@ -842,5 +842,59 @@ DEBU [TSH]       Self re-exec command: tsh [status --format=json]. tsh/aliases.g
 ...
 ```
 
+# Interactive Recording
+
+Within the Teleport cluster the `ssh` and `Kubernetes` recorded sessions are available for listing and replay for authorized users via `tsh`.
```

Comment:
I don't think you need to code-block `ssh` or `Kubernetes` here. Just say "SSH and Kubernetes sessions"


Also, this sentence is pretty long. Maybe we can simplify:

> Recordings for SSH and Kubernetes sessions can be listed with the `tsh` tool.

Reply:
Agreed re: code blocks. Additionally, aren't we replacing the term "SSH Access" with "Server Access"?

Reply:
Updated @zmb3 @alexfornuto